spark/main.go

```diff
@@ -27,21 +27,21 @@ type Service struct {
 
 // JobParams defines the parameters for a Beast job
 type JobParams struct {
-	ClientTag           string
-	ExtraArguments      map[string]interface{}
-	ProjectInputs       []JobSocket
-	ProjectOutputs      []JobSocket
-	ExpectedParallelism int
+	ClientTag           string                 `json:"clientTag"`
+	ExtraArguments      map[string]interface{} `json:"extraArguments"`
+	ProjectInputs       []JobSocket            `json:"projectInputs"`
+	ProjectOutputs      []JobSocket            `json:"projectOutputs"`
+	ExpectedParallelism int                    `json:"expectedParallelism"`
 }
 
 // JobSocket defines the input/output data map
 type JobSocket struct {
 	// Alias: mapping key to be used by a consumer
-	Alias string
+	Alias string `json:"alias"`
 	// DataPath: fully qualified path to actual data, i.e. abfss://..., s3://... etc.
```

Comment:
I think `s3a://` in docs is better than `s3://` because we are using `s3a` scheme in spark, `s3` can be confusing